demisto_sdk/commands/common/tools.py

```diff
@@ -3528,6 +3537,27 @@ def get_pack_paths_from_files(file_paths: Iterable[str]) -> list:
     return list(pack_paths)
 
 
+def replace_incident_to_alerts(value: str) -> str:
+    if not isinstance(value, str):
+        return value
+
+    new_value = value
+    for pattern, replace_with in TABLE_INCIDENT_TO_ALERT.items():
+        new_value = re.sub(pattern, replace_with, new_value)
+    return new_value
+
+
+def replace_alert_to_incident(value: str) -> str:
+    if not isinstance(value, str):
+        return value
+
+    TABLE_ALERT_TO_INCIDENT = {v: k for k, v in TABLE_INCIDENT_TO_ALERT.items()}
+    new_value = value
+    for pattern, replace_with in TABLE_ALERT_TO_INCIDENT.items():
+        new_value = re.sub(pattern, replace_with, new_value)
+    return new_value
```

Comment:
```suggestion
    new_value = value
    for incident, alert in TABLE_INCIDENT_TO_ALERT.items():
        new_value = re.sub(alert, incident, new_value)
    return new_value
```
saves flipping keys/values on each call